### PR TITLE
Fix an issue where elasticat would collect links and drop them withou…

### PR DESCRIPTION
**Pull request by unkempthenry**
…t executing them.

Changes: 

- beacon responses are now added to a queue (previously some where being dropped)

### CODE REVIEW

**Comment by wbooth** [comment]
Looks fine, but begs the question on why the need to introduce Queues if ragdoll does not need them. Does the same issue exist there too?

**Comment by unkempthenry** [comment]
@elegantmoose went through ragdoll a while ago and gave it pretty thorough overhaul.  Queues definitely aren't needed, but a queue of instructions is pretty simple and straightforward. I think there's a pretty good chance we'll want to parallelize these queries someday soon and a queues are friendly to that.  Ragdoll was a handy starting point but I it's OK if these diverge.

File: payloads/elasticat.py [hunk 1]
```diff
@@ -63,27 +65,32 @@ def start(self):
...
         beacon['results'] = results
         body = self._encode_string(json.dumps(beacon))
         resp = requests.post('%s/beacon' % (self.server,), data=body)
         resp.raise_for_status()
-        return json.loads(self._decode_bytes(resp.text))
+        beacon_resp = json.loads(self._decode_bytes(resp.text))
+        self._profile['paw'] = beacon_resp['paw']
+        self.sleep = beacon_resp['sleep']
+        if enqueue_instructions and 'instructions' in beacon_resp:
+            for instruction in json.loads(beacon_resp['instructions']):
```
Comment (wbooth) [inline payloads/elasticat.py:91]: beacon_resp.get('nist', [])

Comment (unkempthenry) [reply payloads/elasticat.py]: 79110db

**Pull request merged**